### ISSUE TITLE
WebAPI/Atlas Release v2.8.0 Planning

### ISSUE THREAD

**Issue by anthonysena** [OHDSI/WebAPI#981]
Hi,

We are planning the v2.8.0 release of WebAPI/Atlas for May 2019 and are posting this thread to raise awareness of the work planned and as a call for contributions to the release. As discussed on the [Atlas/WebAPI Working Group](http://www.ohdsi.org/web/wiki/doku.php?id=projects:workgroups:atlas-webapi) calls,  and the focus of the 2.8 release will be performance optimization for the platform. To that end, here is a high-level list of ideas solicited from the community for the 2.8 release:

- [ ] Expand caching capabilities for WebAPI - look at Redis as a potential option (#1018) @chrisknoll 
- [ ] Make the data sources feature in Atlas fully compatible with information from AchillesWeb and utilize caching for performance. The aim: retire AchillesWeb once this work is complete. @alondhe, @t-abdul-basser, @anthonysena 
- [ ] Vocab search with SOLR and also tunes vocab queries (#580) @fdefalco 
- [ ] Characterization/Treatment Pathways results performance @pavgra 
- [ ] Explore Hierarchy performance

(Tagging folks above based on prior discussions)

In order to submit an idea for a feature, please respond to this thread with the following information by 4/3/2019 so that we can anticipate any potential challenges for incorporating the features:

- A short description of the feature
- The organization or developers that are involved in the implementation
- Impact to current WebAPI/Atlas functionality (if applicable)

After you have responded to this thread, please create a new issue so that the specific feature can be tracked in the V2.8.0 milestone.

Thank you!

**Comment by AnthonyMolinaro** [comment]
**New feature:** Co-occurrence queries for event tables.  These are new Achilles queries for a "recommender" system.  For each event concept_id (condition,drug,procedure,measure,observation), the queries return the top 15 co-occurring concept_ids (in terms of distinct person counts) in  the same domain.

**Devs:** @AnthonyMolinaro 

**Impact to WebAPI/Atlas:** A new tab would have to be added to the UI so that recommendations can be provided to users while building concept sets.

**Comment by anthonysena** [comment]
Closing - this is stale and we will put out a new call for participation in a 2.8 release.
